.npmignore

```diff
@@ -0,0 +1,17 @@
+*.log
+node_modules
+src
+tests
+.github
+dangerfile.js
+.travis.yml
+.gitattributes
+.babelrc
+.eslintrc
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+jestConfig.js
+jestSetup.js
```

Comment:
Do we have a jest setup?

Reply:
Not anymore, it mainly had stuff for react. I didn't include it in this repo.
https://github.com/cerner/terra-aggregate-translations/pull/1/commits/e30c662bfdecaf4554f9cd135bc9552e4d933978